omnisend/src/connection/components/connection-page-layout.js

```diff
@@ -0,0 +1,68 @@
+import ConnectionLogos from './connection-logos';
+import ConnectionFeatures from './connection-features';
+import ConnectionSteps from './connection-steps';
+import { useState } from '@wordpress/element';
```

Comment:
minor: we probably can import react directly instead using this wrapper https://github.com/WordPress/gutenberg/issues/54074#issuecomment-1742726399

Reply:
'react' should be listed in the project's dependencies.
In our case, it is only peerDependencies